docs/dev/check_references.md

```diff
@@ -16,7 +16,7 @@ further_reading:
 
 ## Overview
 
-This page walks you through how to develop an offering on the [**Integrations** page][12] or the [**Marketplace** page][11]. 
+This page walks you through how to develop an offering on the [**Integrations** page][12] or the [**Marketplace** page][9]. 
```

Comment:
Should we call out that this page is specific to the files required when developing? I think it would be helpful to further differentiate between the two.